### ISSUE TITLE
Support multiple view paths

### ISSUE THREAD

**Issue by svenluijten** [svenluijten/artisan-view#41]
If the `views.paths` array contains more than one item, we currently throw an exception alerting the user of the fact that that is not supported. Instead, I'd like to ask the user what path to create the view in. For instance, when `config/views.php` looks like this:

```php
<?php

return [
    'paths' => [
        resource_path('views'),
        resource_path('legacy_views'),
    ];

    //
];
```

It should ask the user if they want to create the view in the `views` or `legacy_views` folder when executing `php artisan make:view`.

**Comment by chrisdicarlo** [comment]
That makes sense, totally agree.

**Comment by svenluijten** [comment]
fixed by #46.
